src/player.cpp

```diff
@@ -884,22 +891,24 @@ void Player::ResetGameObjects() {
 	Main_Data::game_switches = std::make_unique<Game_Switches>();
 	Main_Data::game_switches->SetLowerLimit(lcf::Data::switches.size());
 
-	auto min_var = lcf::Data::system.easyrpg_variable_min_value;
+	auto min_var = (iniVars_min != 0) ? iniVars_min : lcf::Data::system.easyrpg_variable_min_value;
 	if (min_var == 0) {
 		if (Player::IsPatchManiac()) {
 			min_var = std::numeric_limits<Game_Variables::Var_t>::min();
 		} else {
 			min_var = Player::IsRPG2k3() ? Game_Variables::min_2k3 : Game_Variables::min_2k;
 		}
 	}
-	auto max_var = lcf::Data::system.easyrpg_variable_max_value;
+	auto max_var = (iniVars_max != 0) ? iniVars_max : lcf::Data::system.easyrpg_variable_max_value;
 	if (max_var == 0) {
 		if (Player::IsPatchManiac()) {
 			max_var = std::numeric_limits<Game_Variables::Var_t>::max();
 		} else {
 			max_var = Player::IsRPG2k3() ? Game_Variables::max_2k3 : Game_Variables::max_2k;
 		}
 	}
+
```

Comment:
Please remove the 2nd commit. I'm against these new ini flags features. Instead I added in #3209 the setting ``Maniac=2`` to the INI. Is like ``Maniac=1`` but it does not do that dumb range adjustment. (also consistent with your Maniac Patch Flag proposal which will also not adjust the range...)

Reply:
range config is gone